app/views/sections/destination/DestinationWarehouseVatView.scala.html

```diff
@@ -46,14 +48,17 @@
         @govukInput(
             InputViewModel(
                 field = form("value"),
-                label = LabelViewModel(Text(messages("destinationWarehouseVat.heading" , movementScenario.stringValue))).asPageHeading(LabelSize.Large)
+                label = LabelViewModel(Text(messages("destinationWarehouseVat.heading" , movementScenario.stringValue))).asPageHeading(LabelSize.Large),
             )
+            .withHint(HintViewModel(Text(messages("destinationWarehouseVat.hint"))))
             .withWidth(Full)
         )
 
-    @continueOrSaveOrSkip(skipQuestionCall.url)
-
-
+        @if(DestinationWarehouseVatPage.isSkippable(movementScenario)) {
+            @continueOrSaveOrSkip(skipQuestionCall.url)
```

Comment:
Is this using the right skip link?  the copy deck says "Skip this question", but the code says "Skip this question for now" when rendered on screen

Reply:
Good spot. Will update now!